### ISSUE TITLE
Bioshock Infinite - Glossy Sheen/Shine on characters and all objects in game

### ISSUE THREAD

**Issue by JoTechOfficial** [doitsujin/dxvk#4571]
### LOOK AT HAT AND JACKET ON THE RIGHT IN THE PICTURES
I am on Nobara 40 (Linux)

- ### DXVK ON
![1  Shine DXVK ON](https://github.com/user-attachments/assets/db68e9ab-357a-40dc-991a-8e988053d69d)

- ### DXVK OFF
![2  No Shine DXVK OFF](https://github.com/user-attachments/assets/2963bb41-6f44-4ce5-b2ba-db886cb61771)

### Software information
Bioshock Infinite GOG Version (32bit Game)
(Used Heroic Launcher and Ultra settings at 4k)

### System information
- Software Used: Heroic Launcher
- GPU: AMD Radeon RX 7900 XTX
- Driver: Mesa 25.0.0-devel
- (1st Option That I used) Proton version: Experimental
- (2nd Option That I used) Wine version: wine-ge-8-26-x86_64
- DXVK version: 2.5.2

- ### Another Screenshot with DXVK ON
<p align="center">
As you can see Elizabeth doesn't have this problem. 
I've noticed after playing for a bit that sometimes the twins will have a glossy sheen/shine and sometimes they don't. </p>

![Screenshot_20241231_213907](https://github.com/user-attachments/assets/c850de6a-7b47-439d-a63d-847f6b58eeb2)

Log file for proton and wine included as an attachment. I used heroics built in log maker for the game.
[(protonexperimental_dxvk2.5.2)71ZiF1GVqjA4uXoVxRmw9r-lastPlay.log](https://github.com/user-attachments/files/18283805/protonexperimental_dxvk2.5.2.71ZiF1GVqjA4uXoVxRmw9r-lastPlay.log)
[(wine-ge-8-26-x86_64_dxvk2.5.2)71ZiF1GVqjA4uXoVxRmw9r-lastPlay.log](https://github.com/user-attachments/files/18283806/wine-ge-8-26-x86_64_dxvk2.5.2.71ZiF1GVqjA4uXoVxRmw9r-lastPlay.log)

**Comment by doitsujin** [comment]
Is this a recent regression? This was one of the first games that ever worked with DXVK and I have never seen issues like this.

**Comment by JoTechOfficial** [comment]
> Is this a recent regression? This was one of the first games that ever worked with DXVK and I have never seen issues like this.
> 
> Also, does this happen without vkbasalt?

Yes it does still happen. I turned everything off in heroic launcher (in the game settings) besides proton experimental and wine ge tests.

**Comment by Blisto91** [comment]
Fwiw from what i can tell i have not been able to reproduce this issue so far in game. Tested on both a Intel Arc b580 and a AMD rdna2 iGPU.

**Comment by JoTechOfficial** [comment]
> Fwiw from what i can tell i have not been able to reproduce this issue so far in game. Tested on both a Intel Arc b580 and a AMD rdna2 iGPU. This is using the Steam version of the game

I'm using the non steam version. I'm using a drm free gog copy that I bought a long time ago. Heroic game launcher is what I am using for tests.

**Comment by JoTechOfficial** [comment]
![Screenshot_20250101_134324](https://github.com/user-attachments/assets/0b9bafd9-c67a-48a3-ad5e-82e19141d9e2)

Just to make sure I even added my GOG game exe to steam using proton experimental and it's doing the same thing. Seems like steam has preapplied fixes or something when using the steam official version?

**Comment by mbriar** [comment]
> Seems like steam has preapplied fixes or something when using the steam official version?

No, that's extremely unlikely. What is more likely is that it's a driver bug exclusive to RDNA3 GPUs.

**Comment by JoTechOfficial** [comment]
> > Seems like steam has preapplied fixes or something when using the steam official version?
> 
> No, that's extremely unlikely. What is more likely is that it's a driver bug exclusive to RDNA3 GPUs.

Alright bout to download the steam version just to see.

**Comment by mbriar** [comment]
Still will download the native linux port using opengl by default unless you force proton as compat tool though. But i'm sure @Blisto91 was aware of that.

**Comment by Blisto91** [comment]
Ye i made sure it was the Proton version

**Comment by JoTechOfficial** [comment]
![image](https://github.com/user-attachments/assets/bdfc2d96-440f-4769-b442-e1a133c76c7e)
Downloading now and forced proton experimental. I am keeping my settings the same as in heroic launcher with esync, fsync, gamemode, mangohud, and vkbasalt enabled.

**Comment by JoTechOfficial** [comment]
Still same issue even on the steam official version. Maybe it only shows up when on 7000 series gpus? I've noticed some games recently add fixes for the 7000 series so it could be possible that what mbriar is saying is true. Kinda weird cause out of all the bioshocks this is the only one to have this issue. Never had issues in any other games, most of the unreal engine 3 as well.

**Comment by mbriar** [comment]
Driver bugs that trigger only in specific games on some specific GPUs aren't really all that uncommon.

**Comment by mbriar** [comment]
You could try https://github.com/GPUOpen-Drivers/AMDVLK and test if it happens with that as well.

**Comment by JoTechOfficial** [comment]
> Driver bugs that trigger only in specific games on some specific GPUs aren't really all that uncommon.

Just hope there is a fix, love this game. If not i'll just be content. Much appreciated for the advice so far.

**Comment by JoTechOfficial** [comment]
> You could try https://github.com/GPUOpen-Drivers/AMDVLK and test if it happens with that as well.

I'm a noob to linux, only really devoted myself full time to it 2 days ago. I am afraid I'll break something, unless I know how to install properly and be able to uninstall while keeping my current drivers.

**Comment by mbriar** [comment]
> Just hope there is a fix, love this game. If not i'll just be content. Much appreciated for the advice so far.

Driver bugs can be fixed. You can report it here: https://gitlab.freedesktop.org/mesa/mesa/-/issues

**Comment by Blisto91** [comment]
I have a rdna3 GPU in the pile, so i can check it out next time i switch if needed.

**Comment by JoTechOfficial** [comment]
> I have a rdna3 GPU in the pile, so i can check it out next time i switch if needed.

Would be appreciated if you could, if not no worries.

**Comment by JoTechOfficial** [comment]
> > Just hope there is a fix, love this game. If not i'll just be content. Much appreciated for the advice so far.
> 
> Driver bugs can be fixed. You can report it here: https://gitlab.freedesktop.org/mesa/mesa/-/issues

What gpu driver is it using on the times that I didn't have issues? (I am on nobara 40)

**Comment by JoTechOfficial** [comment]
This has been fixed for some reason. Nobara 40 was the issue. Upgrading to 41 ended up fixing the problem... maybe some better packages????

**Comment by mbriar** [comment]
Fedora 41 comes with newer drivers than 40 (Mesa package), so not surprising that it could fix such issues.

**Comment by JoTechOfficial** [comment]
Yeppers bois it's fixed. Sorry about all this :) Thumbs up for all you guys 👍🏻

**Comment by JoTechOfficial** [comment]
Turns out it was not fixed after reboot lol. Could changing themes affect the game someone. Literally the only thing I did different.

**Comment by JoTechOfficial** [comment]
Found out what it was, Mip-map LoD bias. I had it set to -16 and it made the game shiney. Set it to 0 and it fixed it. This was in goverlay which edits mangohud. THIS IS NOT THE FAULT OF DXVK. My deepest apologies
